asv/commands/show.py

```diff
@@ -208,20 +204,20 @@ def get_stat_info(key):
             values = get_stat_info(key)
 
             if key == 'ci_99':
-                values = ["({}, {})".format(util.human_value(x[0], benchmark['unit']),
-                                            util.human_value(x[1], benchmark['unit']))
+                values = [f"({util.human_value(x[0], benchmark['unit'])}, "
+                          f"{util.human_value(x[1], benchmark['unit'])})"
```

Comment:
This is unreadable.

Reply:
This fails with `x is not defined` when i declare the two variables `util.human_value(x[0], benchmark['unit'])` and `util.human_value(x[1], benchmark['unit'])` out of the list

Reply:
Let me do this

Reply:
I created Issue #1193 for this particular bug

Reply:
I'm checking again, and it doesn't make any sense to get the `x is not defined` error when moving the values to a variable. Can you make the update, so I can see the code and the error in the CI please?